content/en/docs/concepts/storage/volumes.md

```diff
@@ -1263,47 +1235,42 @@ spec:
 
 ## Resources
 
-The storage media (Disk, SSD, etc.) of an `emptyDir` volume is determined by the
+The storage media (such as Disk or SSD) of an `emptyDir` volume is determined by the
 medium of the filesystem holding the kubelet root dir (typically
-`/var/lib/kubelet`).  There is no limit on how much space an `emptyDir` or
-`hostPath` volume can consume, and no isolation between Containers or between
-Pods.
+`/var/lib/kubelet`). There is no limit on how much space an `emptyDir` or
+`hostPath` volume can consume, and no isolation between containers or between
+pods.
 
 In the future, we expect that `emptyDir` and `hostPath` volumes will be able to
 request a certain amount of space using a [resource](/docs/concepts/configuration/manage-resources-containers/)
 specification, and to select the type of media to use, for clusters that have
 several media types.
 
-## Out-of-Tree Volume Plugins
+## Out-of-tree volume plugins
 
-The Out-of-tree volume plugins include the Container Storage Interface (CSI)
-and FlexVolume. They enable storage vendors to create custom storage plugins
+The out-of-tree volume plugins include the `Container Storage Interface (CSI)`
+and `FlexVolume`. They enable storage vendors to create custom storage plugins
 without adding them to the Kubernetes repository.
 
-Before the introduction of CSI and FlexVolume, all volume plugins (like
-volume types listed above) were "in-tree" meaning they were built, linked,
-compiled, and shipped with the core Kubernetes binaries and extend the core
-Kubernetes API. This meant that adding a new storage system to Kubernetes (a
-volume plugin) required checking code into the core Kubernetes code repository.
+Previously, all volume plugins were "in-tree". The "in-tree" plugins were built, linked, compiled,
+and shipped with the core Kubernetes binaries. This meant that adding a new storage system to
+Kubernetes (a volume plugin) required checking code into the core Kubernetes code repository.
 
-Both CSI and FlexVolume allow volume plugins to be developed independent of
+Both `CSI` and `FlexVolume` allow volume plugins to be developed independent of
```

Comment:
```suggestion
The `csi` and `flexVolume` volume types both allow storage plugins to be developed independent of
```
?

Reply:
Previously, csi and flexvolume are called volume plugins (line 1251).